src/docker/Dockerfile

```diff
@@ -85,7 +86,7 @@ RUN wget -O /root/task.tar.gz "https://github.com/go-task/task/releases/download
 # Install shellcheck. Get versions from https://github.com/koalaman/shellcheck/releases
 ARG SHELLCHECK_VERSION="0.7.1"
 ENV SHELLCHECK_VERSION=${SHELLCHECK_VERSION}
-# hadolint ignore=DL3003
+# hadolint ignore=DL3003,DL3047
```

Comment:
I don't think we want this ignore here.  We should update `wget` to use the `--progress` flag.

See [here](https://github-wiki-see.page/m/hadolint/hadolint/wiki/DL3047)